gpu_reliability/platforms/gcp.py

```diff
@@ -1,3 +1,5 @@
+import functools
```

Comment:
nit: can you import as `from functools import partial`?

Reply:
Thanks, done.